modules/logging-input-spec-filter-audit-infrastructure.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * logging/performance_reliability/logging-input-spec-filtering.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="logging-input-spec-filter-audit-infrastructure_{context}"]
+= Filtering audit and infrastructure logs inputs by source
+
+You can define the list of `aduit` and `infrastructure` sources to collect logs by using the `input` selector.
```

Comment:
I'm ignoring this change since the writing style will differ from other chapters.

Rephrasing it to: You can define the list of `audit` and `infrastructure` sources to collect the logs by using the `input` selector.